.github/workflows/integration-tests-api-tier-2.yml

```diff
@@ -0,0 +1,71 @@
+name: Integration tests for API - Tier 2
+
+on:
+  pull_request:
```

Comment:
This should be only manual.

Reply:
Could you explain to me why this should be manual?

Reply:
Done